packages/nodes-base/nodes/KoboToolbox/descriptions/Operations.ts

```diff
@@ -0,0 +1,130 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const operations = [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'form',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Retrieve a form definition',
+					},
+				],
+				default: 'get',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'submission',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Query',
+						value: 'query',
+						description: 'Query matching submissions',
+					},
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Get a single submission',
+					},
+					{
+						name: 'Delete',
+						value: 'delete',
```

Comment:
Delete returns an empty array. It should return { success: true }  

![image](https://user-images.githubusercontent.com/16496553/149586284-de08b2df-2d29-46ca-b9b1-c0c22d2fc64e.png)